pangeo_forge_recipes/recipes/__init__.py

```diff
@@ -2,3 +2,24 @@
 from .xarray_zarr import XarrayZarrRecipe
 
 __all__ = ["XarrayZarrRecipe", "HDFReferenceRecipe"]
+
+
+def setup_logging(level: str = "INFO"):
+    """A convenience function that sets up logging for developing and debugging recipes in Jupyter,
+    iPython, or another interactive context.
+
+    :param level: One of (in decreasing level of detail) ``"DEBUG"``, ``"INFO"``, or ``"WARNING"``.
+      Defaults to ``"INFO"``.
+    """
+    import logging
+
+    from rich.logging import RichHandler
+
+    logger = logging.getLogger("pangeo_forge_recipes")
+    if logger.hasHandlers():
+        logger.handlers.clear()
+    logger.setLevel(getattr(logging, level))
```

Comment:
logging level is just an int, if you wanted to make the default level `logging.INFO`.

Reply:
Thanks for jumping in on this review, Alex.

AFAICT, by setting the default level as we currently do, via a default `level="INFO"` in the signature, the function is more self-documenting than if we were to set the default with the `int` equivalent. That is, the default is currently set in a manner that reflects how we'd like users to interact with it.